src/Jupyter/Extensions.cs

```diff
@@ -1,355 +1,355 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
+// Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
 #nullable enable
 
-using System;
-using System.Collections.Generic;
-using System.Collections.Immutable;
 using System.Diagnostics.CodeAnalysis;
-using System.Linq;
 using System.Numerics;
-using System.Text;
-using System.Text.RegularExpressions;
-using Microsoft.Jupyter.Core;
+using Microsoft.Quantum.IQSharp.Common;
 using Microsoft.Quantum.QsCompiler.SyntaxTokens;
 using Microsoft.Quantum.QsCompiler.SyntaxTree;
 using Microsoft.Quantum.Simulation.Common;
-using Microsoft.Quantum.Simulation.Core;
 using Microsoft.Quantum.Simulation.Simulators;
 using Newtonsoft.Json;
 using NumSharp;
 
-namespace Microsoft.Quantum.IQSharp.Jupyter
+namespace Microsoft.Quantum.IQSharp.Jupyter;
+
+/// <summary>
+///      Extension methods to be used with various IQ# and Jupyter objects.
+/// </summary>
+public static class Extensions
 {
     /// <summary>
-    ///      Extension methods to be used with various IQ# and Jupyter objects.
+    ///     Given a configuration source, applies an action if that
+    ///     configuration source defines a value for a particular
+    ///     configuration key.
     /// </summary>
-    public static class Extensions
+    internal static IConfigurationSource ApplyConfiguration<T>(
+        this IConfigurationSource configurationSource,
+        string keyName, Action<T> action
+    )
     {
```

Comment:
(nit) I got an impression that there are 
* relatively small functional changes in this file
* and relatively small cosmetic changes (indentation change) that look like huge changes obscuring the functional changes. 

Consider making cosmetic changes (indentation change) in a separate PR, to track the changes easier.

Reply:
For whitespace only changes, the `?w=1` feature of GitHub can help a lot; for example, https://github.com/microsoft/iqsharp/pull/682/files?w=1 ignores all whitespace-only changes when showing diffs.